package.json

```diff
@@ -8,11 +8,11 @@
     "lint": "jshint index.js lib/*.js"
   },
   "dependencies": {
-    "async": "^1.5.2",
+    "async": "^2.3.0",
     "confit": "^2.0.0",
-    "debug": "^2.1.0",
+    "debug": "^2.6.4",
     "lodash": "^4.12.0",
-    "selenium-webdriver": "^2.53.2",
+    "selenium-webdriver": "2.53.3",
```

Comment:
2.53.3 is the last version in Selenium 2.0 So makes sense to just 2.53.3 as there will be no more version 2.x.x